### PR TITLE
dev-util/emscripten: version bump

### PR DESCRIPTION
**Pull request by timeraider4u**
Package-manager: portage-2.3.3 repoman-2.3.2

### CODE REVIEW

File: dev-util/emscripten-fastcomp/emscripten-fastcomp-1.37.9.ebuild [hunk 1]
```diff
@@ -40,6 +40,8 @@ src_configure() {
 	local mycmakeargs=(
 		# avoid clashes with sys-devel/llvm
 		-DCMAKE_INSTALL_PREFIX="/usr/share/${P}"
+		-DLLVM_OCAML_INSTALL_PATH="/usr/share/${P}/ocaml"
```
Comment (marbre) [inline dev-util/emscripten-fastcomp/emscripten-fastcomp-1.37.9.ebuild]: Can we make this prefix compatible, putting `${ED}` at the beginning of the path?

Comment (timeraider4u) [inline dev-util/emscripten-fastcomp/emscripten-fastcomp-1.37.9.ebuild]: cmake-utils.eclass already adds a prefix in https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/cmake-utils.eclass#n812, see also https://cmake.org/cmake/help/v3.0/variable/CMAKE_INSTALL_PREFIX.html for more details.

Comment (marbre) [inline dev-util/emscripten-fastcomp/emscripten-fastcomp-1.37.9.ebuild]: Thanks, wasn't aware of this :)

**Comment by timeraider4u** [comment]
Thanks for merging!

**Pull request merged**